extensions/Worker.Extensions.Abstractions/src/FixedDelayRetryAttribute.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.Functions.Worker
+{
+    /// <summary>
+    /// Attribute used to mark a function that has an FixedDelay retry policy.
```

Comment:
Consider referring the webjobs version of these attributes and see whether we should use the same documentation/summary
https://github.com/Azure/azure-webjobs-sdk/blob/dev/src/Microsoft.Azure.WebJobs.Host/FixedDelayRetryAttribute.cs#L11